packages/mattermost-redux/src/actions/channel_categories.test.js

```diff
@@ -327,6 +327,89 @@ describe('fetchMyCategories', () => {
         expect(state.entities.channelCategories.byId.category2).toEqual(categories[1]);
         expect(state.entities.channelCategories.orderByTeam[teamId]).toEqual(['category1', 'category2']);
     });
+    test('should update collapse state if it s from websocket', async () => {
```

Comment:
There's a minor typo here since this is the opposite of what we're testing for
```suggestion
    test('should update collapse state if it\'s not from websocket', async () => {
```